### PR TITLE
Update schema to enable panel CM (#575)

### PR DESCRIPTION
**Pull request by Alpana07**
The commit makes required change to enable base and LCD panel CM on Everest systems.
To achieve that whenever the FRU is set for removal,CCIN property needs to be cleared as some sensors are dependent of panel presence and listening for CCIN property of the FRU.
Also post the FRU is replaced, vpd-manager is trigerred to collect VPD of the FRU and restore its CCIN value and present property.

Test:
Ran validator, no new error

Redfish run result-
    {
      "@odata.id": "/redfish/v1/Chassis/chassis/Assembly#/Assemblies/2",
      "@odata.type": "#Assembly.v1_3_0.AssemblyData",
      "Location": {
        "PartLocation": {
          "ServiceLabel": "UABCC.ND0.0000111-D0"
        }
      },
      "LocationIndicatorActive": false,
      "MemberId": "2",
      "Model": "51F4",
      "Name": "panel0",
      "Oem": {
        "OpenBMC": {
          "@odata.type": "#OemAssembly.v1_0_0.Assembly",
          "ReadyToRemove": true
        }
      },

journal-
May 01 14:31:59 ever8bm vpd-manager[1081]: Manager called to delete vpd for fru: /system/chassis/motherboard/dasd_backplane/panel0 May 01 14:31:59 ever8bm vpd-manager[1081]: Unbinding device- May 01 14:31:59 ever8bm vpd-manager[1889]: sh: line 1: echo: write error: No such device May 01 14:32:06 ever8bm entity-manager[534]: Inventory Removed: Storm King Panel May 01 14:32:06 ever8bm kernel: i2c i2c-29: delete_device: Deleting device dps310 at 0x76 May 01 14:32:06 ever8bm virtual-sensor[359]: Removed a virtual sensor: Ambient_Virtual_Temp May 01 14:32:06 ever8bm kernel: i2c i2c-29: delete_device: Deleting device si7020 at 0x40 May 01 14:32:06 ever8bm kernel: i2c i2c-29: delete_device: Deleting device tmp75 at 0x48

May 01 14:32:51 ever8bm vpd-manager[1081]: Manager called to collect vpd for fru: /system/chassis/motherboard/dasd_backplane/panel0
May 01 14:32:51 ever8bm kernel: at24 29-0050: 4096 byte 24c32 EEPROM, writable, 1 bytes/write
May 01 14:32:51 ever8bm systemd[1]: Started IPZ format VPD Parser service for FRU sys/devices/platform/ahb/ahb:apb/ahb:apb:bus@1e78a000/1e78a780.i2c-bus/i2c-14/i2c-29/29-0050/29-005017.
May 01 14:32:52 ever8bm systemd[1]: ibm-vpd-parser@sys-devices-platform-ahb-ahb:apb-ahb:apb:bus\x401e78a000-1e78a780.i2c\x2dbus-i2c\x2d14-i2c\x2d29-29\x2d0050-29\x2d005017.service: Deactivated successfully.
May 01 14:32:59 ever8bm entity-manager[534]: Inventory Added: Storm King Panel

Change-Id: I117ca2b27826c672e3dbdec631fb1bc16dc2494c

### CODE REVIEW

File: redfish-core/lib/assembly.hpp [hunk 1]
```diff
@@ -687,6 +701,42 @@ inline void setAssemblyLocationIndicators(
...
+                // exposed by vpd-manager to clear CCIN and set present
+                // property as false for the FRU.
+                // readytoremove as false implies FRU has been replaced. Set
+                // action as "CollectFRUVPD". This is the api exposed by
+                // vpd-manager to recollect vpd for a given FRU.
+                std::string action = (readytoremove.value()) ? "deleteFRUVPD"
+                                                             : "CollectFRUVPD";
+
+                crow::connections::systemBus->async_method_call(
+                    [asyncResp, action](const boost::system::error_code ec) {
```
Comment (baemyung) [inline redfish-core/lib/assembly.hpp:726]: `const boost::system::error_code& ec`

File: redfish-core/lib/assembly.hpp [hunk 2]
```diff
@@ -256,24 +256,38 @@ void getAssemblyPresence(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
...
+         assembly](const boost::system::error_code& ec, const bool value) {
         if (ec)
         {
             BMCWEB_LOG_ERROR("DBUS response error: {}", ec.value());
             messages::internalError(asyncResp->res);
             return;
         }
 
+        nlohmann::json& array = asyncResp->res.jsonValue["Assemblies"];
+        nlohmann::json& data = array.at(assemblyIndex);
```
[the remark this answers is not in the document]
Comment (Alpana07) [reply redfish-core/lib/assembly.hpp:274]: ok

File: redfish-core/lib/assembly.hpp (hunk 1 above)
Comment (Alpana07) [reply redfish-core/lib/assembly.hpp:726]: ok

**Review by baemyung — APPROVED** [review]
It looks good to me

**Pull request merged**